scss/_patterns_code-snippet.scss

```diff
@@ -1,6 +1,7 @@
 @import 'settings';
 
 $color-snippet-heading-bg: rgba($color-x-dark, 0.08);
+$header-v-spacing: $spv-inner--x-small--scaleable;
```

Comment:
Such variables are global in whole Vanilla, would be good to make the name prefixed by component name to make sure what it refers to.

Reply:
@sowasred2012 This doesn't seem to be addressed.